rrat/templates/home.html

```diff
@@ -5,12 +5,80 @@
 {% endblock title %}
 
 {% block header %}
-  Home
+  <!-- No title --> 
 {% endblock header %}
 
-
 {% block content %}
-  <section class="px-4 py-6 sm:px-6 lg:px-8 bg-white rounded-md shadow-lg">
-  <h2 class="text-3xl/loose font-bold tracking-tight text-indigo-950">Welcome!</h2>
+
+  <!-- Hero Section -->
+  <section class="hero-section bg-indigo-900 text-white py-16 mt-0">
+    <div class="container mx-auto text-center">
```

Comment:
update this class on line 15 to the following:
```
container mx-auto text-center flex items-center flex-col
```
This will get that text centered